packages/react-native-compat/module/NativeRNWalletConnectModule.ts

```diff
@@ -9,4 +9,4 @@ export interface Spec extends TurboModule {
   };
 }
 
-export default TurboModuleRegistry.getEnforcing<Spec>("RNWalletConnectModule");
+export default TurboModuleRegistry.get<Spec>("RNWalletConnectModule");
```

Comment:
would this be a reaking change for older versions? Can we provide both conditionally somehow? or thats not required

Reply:
no, its not a breaking change and it only affects projects using the new RN architecture (probably 0 now)